projects/sport-data-valley/manual_seeds/rheuma_responses.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require_relative 'helpers/people_helper'
+
+srand(123)
+
+START_DATE = Time.zone.parse('2021-11-24').beginning_of_day
+END_DATE = START_DATE + 3.weeks
+N_PARTICIPANTS = 5
+
+def range_0_10
+  rand(0.0..10.0).round(1)
+end
+
+# Returns a value in between 0 and 24 in increments of 0.5
+def n_hours
+  hours = rand(24*2 + 1)
+  hours * 0.5
+end
+
+def create_rheuma_responses(person, date)
+  return if rand > 0.9
+
+  protocol_name = 'daily_protocol_rheumatism'
+  puts "Creating responses for #{protocol_name}"
+  protocol = Protocol.find_by(name: protocol_name)
+  raise "Error: protocol named #{protocol_name} not found" unless protocol
+
+  protsub = ProtocolSubscription.find_by(person: person, protocol: protocol)
+  protsub ||= ProtocolSubscription.new(person: person, protocol: protocol)
+
+  protsub.state = ProtocolSubscription::COMPLETED_STATE
+  protsub.start_date = Time.new(2016).in_time_zone
+  protsub.end_date = Time.new(2017).in_time_zone
+  protsub.save!
+
+  questionnaire_name = 'Dagelijkse vragenlijst reuma'
+  questionnaire_id = Questionnaire.find_by(name: questionnaire_name).id
+  measurement = protocol.measurements.where(questionnaire_id: questionnaire_id).first
+
+  protsub_id = protsub.id
+  measurement_id = measurement.id
+  filled_out_by_id = protsub.person.id
+  filled_out_for_id = protsub.filling_out_for.id
+
+  Response.where(protocol_subscription_id: protsub_id, measurement_id: measurement_id).each do |response|
+    ResponseContent.find(response.content).destroy if response.content.present?
+    response.destroy
+  end
+
+  response_params = {}
+  response_params[:protocol_subscription_id] = protsub_id
+  response_params[:measurement_id] = measurement_id
+  response_params[:filled_out_by_id] = filled_out_by_id
+  response_params[:filled_out_for_id] = filled_out_for_id
+
+  response_params[:open_from] = date - 1.minute
+  response_params[:opened_at] = date
+  response_params[:completed_at] = date + 1.minute
+
+  content = {}
+  content[:v1] = range_0_10
```

Comment:
Btw I think this generates a number, but it's important to note that what is stored in the DB is always a string.

e.g.:
```ruby
{"locale"=>"nl",
 "v1"=>"5.8",
 "v2"=>"Ja<br />Zo ja, waar denkt u dat dit door kwam?",
 "v2_ja_br_zo_ja_waar_denkt_u_dat_dit_door_kwam_text"=>"geen idee",
 "v3"=>"2.8",
 "v4"=>"7.4",
 "v5"=>"4",
 "v6"=>"2",
 "v7"=>"5.5",
 "v8"=>"7",
 "v1_timing"=>"37793",
 "v2_nee_timing"=>"1648",
 "v3_timing"=>"1132",
 "v4_timing"=>"1046",
 "v5_timing"=>"1558",
 "v6_timing"=>"1811",
 "v7_timing"=>"1257",
 "v8_timing"=>"3654",
 "v2_ja_br_zo_ja_waar_denkt_u_dat_dit_door_kwam_timing"=>"82",
 "v2_ja_br_zo_ja_waar_denkt_u_dat_dit_door_kwam_text_timing"=>"3961"}
```

So for this to be accurate all the numbers should be strings I think.

And don't mind the `_timing` values. You don't have to add those. We don't use them for anything currently.

Reply:
that function adds .to_s